### PR TITLE
Add CORS headers for GeoGirafe demo

### PR DESCRIPTION
**Pull request by llienher**

### CODE REVIEW

File: geoportal/vars.yaml [hunk 1]
```diff
@@ -868,6 +868,9 @@ vars:
         - https://127.0.0.1:3000
         - https://127.0.0.1:3001
         - '*'
+        # GeoGirafe
+        - 'https://app.localhost:8080'
+        - 'https://demo.geomapfish.dev'
```
Comment (sbrunner) [inline geoportal/vars.yaml]: Don't use unneeded quote
```suggestion
        - https://app.localhost:8080
        - https://demo.geomapfish.dev
```

File: geoportal/vars.yaml [hunk 2]
```diff
@@ -868,6 +868,9 @@ vars:
         - https://127.0.0.1:3000
         - https://127.0.0.1:3001
         - '*'
+        # GeoGirafe
+        - 'https://app.localhost:8080'
```
Comment (sbrunner) [inline geoportal/vars.yaml]: What is this URL?

Comment (llienher) [reply geoportal/vars.yaml]: Their localhost demo url

Comment (sbrunner) [reply geoportal/vars.yaml]: It's created by docker compose?

Comment (llienher) [reply geoportal/vars.yaml]: No, it is created with vitejs. Why ?

Comment (sbrunner) [reply geoportal/vars.yaml]: I'm relay surprise to see the `app.` prefix!

Comment (llienher) [reply geoportal/vars.yaml]: Issues with single-label domain and cookies, not something I saw in GMF, but it is what they figured out in their localhost demo ...

**Review by sbrunner — APPROVED** [review]

**Pull request merged**